website/addons/wiki/static/pagedown-ace/Markdown.Editor.js

```diff
@@ -2441,5 +2482,22 @@ var Range = ace.require('ace/range').Range;
         chunk.skipLines(2, 1, true);
     }
 
+    commandProto.helpPopUp = function (chunk, postProcessing){
+      bootbox.dialog({
+          title: 'Wiki Syntax Help',
+          message:
+              '<p>Wiki uses the Markdown snytax. This gives you many options, but can be very simple as well. ' +
+              'To see more information and examples go to our <a href="http://help.osf.io/m/collaborating/l/524109-using-the-wiki">guides</a>' +
+              '</p>'
+      });
+    }
+
 
 })();
+//<button type=button class="btn osf-search-btn" data-toggle="modal" data-target="#search-help-modal"><i class="fa fa-question fa-lg"></i></button>

```

Comment:
No commented out code here (there are some lines I skipped, so be sure to check everywhere).